src/Data/YAML/Aeson.hs

```diff
@@ -141,3 +160,58 @@ scalarToValue (Y.SFloat x)   = Just $! J.Number (realToFrac x)
 scalarToValue (Y.SInt i)     = Just $! J.Number (fromInteger i)
 scalarToValue (SStr t)       = Just $! J.String t
 scalarToValue (SUnknown _ _) = Nothing
+
+
+-- | Equivalent to the fuction Data.ByteString.toStrict.
+-- O(n) Convert a lazy ByteString into a strict ByteString.
+{-# INLINE bsToStrict #-}
+bsToStrict :: BS.L.ByteString -> BS.ByteString
+#if MIN_VERSION_bytestring(0,10,0)
+bsToStrict = BS.L.toStrict
+#else
+bsToStrict = BS.concat . BS.L.toChunks
+#endif
+
+-- | @since 0.2.0
+instance ToYAML J.Value where
```

Comment:
Orphan instance: instance ToYAML Value
Shall I move this to HsYAML?

Reply:
No; that would require to have `HsYAML` depend on `aeson` which would then make having a separate HsYAML-aeson package pointless... :-)

simply add `{-# OPTIONS_GHC -fno-warn-orphans #-}` to the top of the file